### PR TITLE
Adding release v0.2 latest updates

### PR DESCRIPTION
**Pull request by IslamHeggy**
This pull request includes the following changes

-  A new packer template for creating an AMI (Created by Nour)
- Enhancing README by removing outdated instructions and provide instructions for new things
- Fixing a bug in the workflow

### CODE REVIEW

File: .github/workflows/buildandrelease.yml [hunk 1]
```diff
@@ -45,10 +55,9 @@ jobs:
         timeout: 10000s
         command_timeout: 10000s
         script: |
-           cd /home/azureuser/git-build-dir &&  git clone https://github.com/k8-proxy/s-k8-releases.git
-           cd /home/azureuser/git-build-dir/s-k8-releases && git clone --recursive https://github.com/filetrust/k8-reverse-proxy.git
-           tar czvf revproxy.tar.gz -C /home/azureuser/git-build-dir/s-k8-releases/k8-reverse-proxy/ /home/azureuser/git-build-dir/s-k8-releases/reverse-proxy-icap-docker/
-           packer build -force box.json 
+           cd ${{ env.BUILD_DIR }}/s-k8-releases && git clone --recursive git@github.com:NourEddineX/k8-reverse-proxy.git
```
Comment (NourEddineX) [inline .github/workflows/buildandrelease.yml:58]: please use k8-proxy/k8-reverse-proxy:develop , no need to use my fork anymore

File: .github/workflows/buildandrelease.yml [hunk 2]
```diff
@@ -45,10 +55,9 @@ jobs:
...
+           tar czvf revproxy.tar.gz -C ${{ env.BUILD_DIR }}/s-k8-releases/k8-reverse-proxy/stable-src/ .
```
Comment (NourEddineX) [inline .github/workflows/buildandrelease.yml:59]: Please add this line to the README for manual builds

Comment (IslamHeggy) [reply .github/workflows/buildandrelease.yml:59]: Ah yes forgot to revert it back after the edit.

Updated.

File: .github/workflows/buildandrelease.yml (hunk 1 above)
Comment (IslamHeggy) [reply .github/workflows/buildandrelease.yml:58]: Updated.

File: .github/workflows/buildandrelease.yml [hunk 3]
```diff
@@ -1,41 +1,51 @@
...
-      - 'diagrams/**'    
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:   
   build-and-release:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
+    # Steps represent a sequence of taskdjuss that will be executed as part of the job
```
Comment (NourEddineX) [inline .github/workflows/buildandrelease.yml:22]: typo?

Comment (IslamHeggy) [reply .github/workflows/buildandrelease.yml]: Ahh yes fixed it

**Review by NourEddineX — APPROVED** [review]

**Review by DinisCruz — APPROVED** [review]
This is really good 👍

**Pull request merged**